src/doom/am_map.c

```diff
@@ -1882,6 +1866,9 @@ void AM_drawPlayers(void)
 
     for (i=0;i<MAXPLAYERS;i++)
     {
+    // [crispy] interpolate other player arrows angle
```

Comment:
Indentation is a bit messed up throughout this PR. If you could fix these issues, I'd be all fine to merge it. Thanks!

Reply:
Oops, yes, my oversight - was looking up in the code while should look down. 

I've also dropped all Lee's comments with dates only while replacing `consoleplayer` with `displayplayer` (like `// killough 4/25/98`). I belive, they are aren't informative, but all major comments of Lee and Jim are strictly on place.